lib/safer_rails_console/rails_version.rb

```diff
@@ -40,6 +40,19 @@ def six_or_above?
         @is_six_or_above = SaferRailsConsole::RailsVersion::RAILS_VERSION >= ::Gem::Version.new('6.0.0') if @is_six_or_above.nil?
         @is_six_or_above
       end
+
```

Comment:
Do we still we need any of the changes in this file since we removed the version conditional in lib/safer_rails_console/patches/sandbox/auto_rollback.rb?

Reply:
You don't seem to be using the `#six_zero?` method either. I added the methods as part of the scaffolding for Rails 7 support.

Reply:
We removed a bunch of code from this file including the `#six_zero?` method as part of #43. Hopefully we can avoid lots of Rails version specific logic on more modern versions of Rails.

Reply:
I just rebased and cleaned up merge conflicts. I removed the extraneous methods.